manifests/m/madshi/eac3to/3.36/madshi.eac3to.locale.en-US.yaml

```
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: madshi.eac3to
+PackageVersion: "3.36"
+PackageLocale: en-US
+Publisher: madshi.net
+PublisherUrl: http://madshi.net/
+Author: madshi.net
+PackageName: eac3to
+PackageUrl: https://forum.doom9.org/showthread.php?t=125966
+License: Proprietary
+ShortDescription: An audio conversion tool.
+InstallationNotes: |
+  This is a commandline "loose executable" file, but currently can't be launched via terminal.
+  Please navigate to folder "%LOCALAPPDATA%/Microsoft/WinGet/Links" and launch manually.

```

```suggestion
  Please navigate to the install folder and launch manually.
```
Not all installs support the links directory, and users can change where the directory is created. Please avoid putting paths into install notes